app/models/cost_query.rb

```diff
@@ -22,12 +22,12 @@ class CostQuery < Report
 
   User.before_destroy do |user|
     CostQuery.delete_all ['user_id = ? AND is_public = ?', user.id, false]
-    CostQuery.update_all ['user_id = ?', DeletedUser.first.id], ['user_id = ?', user.id]
+    CostQuery.where(['user_id = ?', user.id]).update_all ['user_id = ?', DeletedUser.first.id]
 
     max_query_id = 0
-    while((current_queries = CostQuery.all(limit: 1000,
-                                           conditions: ["id > ?", max_query_id],
-                                           order: "id ASC")).size > 0) do
+    while((current_queries = CostQuery.limit(1000)
+                             .where(["id > ?", max_query_id])
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.